nextcloud/mail
# feat: One-click unsubscribe

Fixes https://github.com/nextcloud/mail/issues/5387
Close https://github.com/nextcloud/mail/issues/5387

Refinement of https://github.com/nextcloud/mail/pull/8395 for services that support one-click unsubscribe.

https://datatracker.ietf.org/doc/html/rfc8058

## How to test

1) Set up an account in Mail
2) Open a message from a mailing list (it has to have `List-Unsubscribe: <http…>` and `List-Unsubscribe-Post=List-Unsubscribe=One-Click` headers)
3) See the *Unsubscribe* button
4) Click the *Unsubscribe* button and see a confirmation
